tests/integration_tests/test_rdflib/test_logical_types.py

```diff
@@ -274,61 +276,60 @@ def test_rdflib_logical_matrix(case: Case) -> None:
         list(case.parser(io.BytesIO(data), logical_type_strict=case.strict))
 
 
+rdparse = importlib.import_module("pyjelly.integrations.rdflib.parse")
+file_path = Path(getattr(rdparse, "__file__", ""))
+IS_COMPILED = file_path.suffix in {".so", ".pyd"}
```

Comment:
This condition is different from `test_lookup_encoder.py`, where `.dll` is also present. Why?